// Copyright 2022 Google LLC
//
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
//      http://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.

// This API is experimental and under-development.

syntax = "proto3";

package sax.server.internal;

message SyncRequest {
  // Monotonically increasing sequence number.
  int64 seqno = 1;
  // Message content.
  string text = 2;
}

message SyncResponse {}

service MultiHostSyncService {
  // Updates the server with a sync message.
  rpc Sync(SyncRequest) returns (SyncResponse);
}
